lib/model.js

```diff
@@ -1236,7 +1236,7 @@ Model.prototype.findById = function(param, options) {
     return Promise.resolve(null);
   }
 
-  options = options || {};
+  options = Utils._.clone(options) || {};
```

Comment:
Please use `optClone` like the other methods instead, it preserves buffers, special instances, etc.